frontend/app/src/components/profitloss/ReportMissingPrices.vue

```diff
@@ -73,6 +77,11 @@ export default defineComponent({
   setup(props) {
     const { items } = toRefs(props);
     const prices = ref<HistoricalPrice[]>([]);
+    const errorMessages: Ref<{ [key: string]: string[] }> = ref({});
```

Comment:
At some point, it will probably make sense to start migrating the validation to [vuelidated](https://vuelidate-next.netlify.app/).

I used it for the website and it seems nice, it also has proper support for mixing frontend/backend validations for the messages.

Reply:
I used something similar in the past [vee validate](https://vee-validate.logaretm.com/v2). But yeah we must get rid all of this manual validation 😆